pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
This ordering does not match the (alphanum) ordering in KubeletServer.  Is this intentional?